### PR TITLE
Announce paper release

### PR DESCRIPTION
**Pull request by WoosukKwon**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -104,3 +105,17 @@ For details, check out our [blog post](https://vllm.ai).
...
+If you use vLLM for your research, please cite our [paper](https://arxiv.org/abs/2309.06180):
+```bibtex
+@misc{kwon2023efficient,
+      title={Efficient Memory Management for Large Language Model Serving with PagedAttention}, 
+      author={Woosuk Kwon and Zhuohan Li and Siyuan Zhuang and Ying Sheng and Lianmin Zheng and Cody Hao Yu and Joseph E. Gonzalez and Hao Zhang and Ion Stoica},
+      year={2023},
+      eprint={2309.06180},
+      archivePrefix={arXiv},
+      primaryClass={cs.LG}
+}
```
Comment (zhuohan123) [inline README.md:120]: Can you change the bib to cite our SOSP version?

**Review by zhuohan123 — COMMENTED** [review]
Please also add a link in our doc. Thanks!

File: README.md (hunk 1 above)
Comment (WoosukKwon) [reply README.md:120]: You mean without the `pages` entry?

Comment (zhuohan123) [reply README.md:120]: Yes, just keep `booktitle` and `year`, for example:

```suggestion
@misc{kwon2023efficient,
      title={Efficient Memory Management for Large Language Model Serving with PagedAttention}, 
      author={Woosuk Kwon and Zhuohan Li and Siyuan Zhuang and Ying Sheng and Lianmin Zheng and Cody Hao Yu and Joseph E. Gonzalez and Hao Zhang and Ion Stoica},
      booktitle={Proceedings of the ACM SIGOPS 29th Symposium on Operating Systems Principles (SOSP'23)},
      year={2023},
}
```

Comment (WoosukKwon) [reply README.md]: Done!

**Review by zhuohan123 — APPROVED** [review]
LGTM!

**Pull request merged**